test/lib/test/config_test.exs

```diff
@@ -0,0 +1,17 @@
+# Copyright(c) 2015-2020 ACCESS CO., LTD. All rights reserved.
+
+Antikythera.Test.Config.init()
+
+defmodule Antikythera.Test.ConfigTest do
+  use Croma.TestCase
+
+  test "blackbox_test_secret/0 should read BLACKBOX_TEST_SECRET_JSON environment variable" do
+    env_var_name = "BLACKBOX_TEST_SECRET_JSON"
+    original = System.get_env(env_var_name)
+    System.put_env(env_var_name, "{\"foo\": \"bar\"}")
+    assert Antikythera.Test.Config.blackbox_test_secret()["foo"] == "bar"
+    if original != nil do
+      System.put_env(env_var_name, original)
+    end
```

Comment:
I think it's better to unset `BLACKBOX_TEST_SECRET_JSON` if this environment variable was originally undefined.

```suggestion
    case original do
      nil -> System.delete_env(env_var_name)
      v   -> System.put_env(env_var_name, v)
    end
```

Reply:
I agree with you.
Fixed at https://github.com/access-company/antikythera/pull/119/commits/6605e60b5bf1b6cd4efd0a15b049289d485500a2